conf/drupal/config/core.entity_form_display.node.binder.default.yml

```diff
@@ -85,6 +86,7 @@ third_party_settings:
         - field_data_resource_type
         - field_data_format
         - field_data_topic
+        - field_data_hub_search_content
```

Comment:
@sagar-lcm After discussing with Joe and Adam, we would like the field name to be more generic. Could we rename the field to `field_data_search_content`?

Reply:
Fixed.